setup.py

```diff
@@ -1,7 +1,6 @@
 #!/usr/bin/env python
-# Copyright (c) 2017 "Shopify inc." All rights reserved.
-# Use of this source code is governed by a BSD-style license that can be
-# found in the LICENSE file.
+# Copyright (c) 2017 "Shopify inc." All rights reserved. Use of this source code is governed by a BSD-style license
```

Comment:
Sorry for nit-picking again. I think the line wrap should be after `All rights reserved.`